docs/content/configuration.md

```diff
@@ -371,19 +371,28 @@ OPA will authenticate using a bearer token obtained through the OAuth2 [client c
+| `services[_].credentials.oauth2.grant_type`                         | `string`   | No       | Defaults to `client_credentials`.                                                                                                                                                                           |
+| `services[_].credentials.oauth2.client_id`                          | `string`   | No       | The client ID to use for authentication.                                                                                                                                                                    |
+| `services[_].credentials.oauth2.signing_key`                        | `string`   | No       | Reference to private key used for signing the JWT. Required if `aws_kms_key` is not provided                                                                                                                |
+| `services[_].credentials.oauth2.thumbprint`                         | `string`   | No       | Certificate thumbprint to use for x5t header generation.                                                                                                                                                    |
+| `services[_].credentials.oauth2.additional_claims`                  | `map`      | No       | Map of claims to include in the JWT (see notes below)                                                                                                                                                       |
+| `services[_].credentials.oauth2.include_jti_claim`                  | `bool`     | No       | Include a uniquely generated `jti` claim in any issued JWT                                                                                                                                                  |
+| `services[_].credentials.oauth2.scopes`                             | `[]string` | No       | Optional list of scopes to request for the token.                                                                                                                                                           |
+| `services[_].credentials.oauth2.aws_kms_key.name`                   | `string`   | No       | To specify a KMS key, use its key ID, key ARN, alias name, or alias ARN. Required only for signing with AWS KMS.                                                                                            |
+| `services[_].credentials.oauth2.aws_kms_key.algorithm`              | `string`   | No       | Specifies the signing algorithm used by the key `aws_kms_key.name` `(ECDSA_SHA_256, ECDSA_SHA_384 or ECDSA_SHA_512)`. Required only for signing with AWS KMS.                                               |
```

Comment:
What do you think about replacing `aws_kms_key.name` to `aws_kms.key` and `aws_kms_key.algorithm` to `aws_kms.algorithm`? Seems more future-proof if we need to add more fields for `aws_kms`.

Reply:
Done